src/main/java/com/webapp/TextBook/repository/data_access/BookCopy.java

```diff
@@ -0,0 +1,551 @@
+package com.webapp.TextBook.repository.data_access;
+
+import com.webapp.TextBook.repository.DataAccessConversion;
+
+import java.math.BigDecimal;
+import java.util.Date;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+
+/**
+ * <h1>BookCopy</h1>
+ * <h2>Type: Class</h2>
+ *
+ * <p>Structure to hold information relating to a BookCopy and it's state.
+ * These values are determined and fully defined from the business context and are explicated within
+ * the FRD and traceability document.</p>
+ *
+ */
+
+@Entity
+public class BookCopy implements DataAccessConversion {
+
+    /**
+     * <p>
+     * Generates a unique ID for BookCopy.
+     * Usage: Hold primary key value for a unique BookCopy instance.
+     * </p>
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID _ID;
+
+    /**
+     * <p>
+     * String variable to hold the book code representing the department and course number.
+     * Usage: Attribute in creation of BookCopy for ORM conversion of Oracle's DBO into DA POJO.
+     * </p>
```

Comment:
what is a barcode?
what is a bookcode?
what is an edition year? ....

This should be added (don't need to re-define the wheel but a  >1 sentence summary would be needed!)

Reply:
Where would this need to be explained more, the variables themselves are mostly self-explanatory. There's not much more to add description wise.